config.mk

```diff
@@ -5,3 +5,5 @@ PACKAGES_LIST_DEFAULT=default backbone
 OPENWRT_SRC=git://git.openwrt.org/15.05/openwrt.git
 OPENWRT_COMMIT=64e116779c0f7da6d98068b8e7c50f528c8a91f2
```

Comment:
@lynxis changing the way of specifying the OpenWRT-commit to be used is not the intention of the issue